src/bp-core/bp-core-catchuri.php

```diff
@@ -1157,6 +1157,16 @@ function bp_private_network_template_redirect() {
 		$id                  = ( ! empty( $id ) ) ? $id : 0;
 		$activate            = ( bp_is_activation_page() && ( '' !== bp_get_current_activation_key() || isset( $_GET['activated'] ) ) ) ? true : false;
 
+		/**
+		 * Filter to check woocommerce registration is enable or not.
+		 *
+		 * @param bool   $validated 	True if message is not valid, false otherwise.
+		 * @param string $id 			Current page ID.
+		 *
+		 * @return bool true if woocommerce registration is enable.
+		 */
+		$is_enable_wc_registration = apply_filters( 'bb_is_enable_woocommerce_myaccount_registration', false, $id );
```

Comment:
this filter should be like `bb_is_enable_registration` or `bb_is_enable_3rd_party_registration`

Reply:
@dipskakadiya Filter name changed. Please check and review